h/static/styles/partials-v2/_nav-bar.scss

```diff
@@ -23,7 +25,6 @@
 }
 
 .nav-bar__logo {
-  margin-left: 30px;
```

Comment:
The left and right margins were on the first and last children of the nav bar header, I've moved them onto the container element instead.